cap-ci/pipeline.template

```diff
@@ -229,6 +343,36 @@ jobs:
             {{- print $import_aks }}
             {{- end }}
             {{- print $deploy }}
+  {{- if $config.create_k8s }}
+  ensure:
+    do:
+      - task: delete-k8s
+        privileged: true
+        timeout: 2h30m
+        input_mapping:
+          pool.kube-hosts: {{$Backend}}-pool.kube-hosts
+        config:
+          platform: linux
+          image_resource:
+            type: registry-image
+            source:
+              repository: splatform/catapult
+          inputs:
+          - name: catapult
+          - name: pool.kube-hosts
+          params:
+            BACKEND: {{$Backend}}
+            QUIET_OUTPUT: true
+            DEFAULT_STACK: cflinuxfs3
```

Comment:
Same here, missing `DOWNLOAD_CATAPULT_DEPS: false`.

This job should be templated and not repeated, or better, have only 1 job in a reconciler pipeline.

Reply:
> Same here, missing `DOWNLOAD_CATAPULT_DEPS: false`.
> 
I guess you reviewed some outdated commit. It is present in next line (364)

> This job should be templated and not repeated, or better, have only 1 job in a reconciler pipeline.

This is not a job. It is a task. We can only handle templating once we know how we will be doing it i.e. post https://jira.suse.com/browse/CAP-1354